server/app/api/v1/endpoints/calls.py

```
@@ -1257,6 +1258,214 @@ async def bulk_upload_calls(
+    """
+    Download call recording by proxying through VAPI API.
+    
+    Args:
+        call_id: Call ID
+        organization: Current organization
+        db: Database session
+        
+    Returns:
+        StreamingResponse: Audio file stream with proper headers
+    """
+    logger.info(f"=== DOWNLOAD RECORDING CALLED ===")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)